tests/test_convert.py

```diff
@@ -823,6 +828,62 @@ def test_convert_cellprofiler_sqlite_pycytominer_merge(
     assert pycytominer_table.shape == cytotable_table.shape
 
 
+def test_sqlite_mixed_type_query_to_parquet(
+    get_tempdir: str, example_sqlite_mixed_types_database: str
+):
+    """
+    Testing _sqlite_mixed_type_query_to_parquet
+    """
+
+    result_filepath = f"{get_tempdir}/example_mixed_types_tbl_a.parquet"
+    table_name = "tbl_a"
+
+    try:
+        # attempt to read the data using DuckDB
+        result = _duckdb_reader().execute(
+            f"""COPY (
+                select * from sqlite_scan('{example_sqlite_mixed_types_database}','{table_name}')
+                LIMIT 2 OFFSET 0
+                ) TO '{result_filepath}'
+                (FORMAT PARQUET)
+            """
+        )
+    except duckdb.Error as duckdb_exc:
+        # if we see a mismatched type error
+        # run a more nuanced query through sqlite
+        # to handle the mixed types
+        if "Mismatch Type Error" in str(duckdb_exc):
+            result = _sqlite_mixed_type_query_to_parquet(
+                source_path=example_sqlite_mixed_types_database,
+                table_name=table_name,
+                chunk_size=2,
+                offset=0,
+                result_filepath=result_filepath,
+            )
```

Comment:
Same comment here as before regarding using `duckdb.TypeMismatchException` over its ancestor `duckdb.Error`.

Reply:
Similar to the previous comment, leaving this as-is for right now.